include/albatross/models/sparse_gp.h

```diff
@@ -202,14 +224,15 @@ class SparseGaussianProcessRegression
      *
      *     v = L^-T B^-1 P * A^-1 y
      */
-    Eigen::VectorXd A_sqrt = A.array().sqrt();
-    Eigen::MatrixXd RtR = A_sqrt.asDiagonal().inverse() * P.transpose();
+    auto A_llt = A.llt();
+    Eigen::MatrixXd Pt = P.transpose();
+    Eigen::MatrixXd RtR = A_llt.matrixL().llt().solve(Pt);
```

Comment:
Is this correct? It should be `A^-1/2`, right?

Reply:
Yeah the matrixL part is the square root, could make that more obvious with naming.